harvester/exceptions.py

```diff
@@ -47,6 +47,10 @@ class CompareException(HarvestCriticalException):
     pass
 
 
+class SendNotificationException(HarvestCriticalException):
+    pass
```

Comment:
We should probably rename the `HarvestCriticalException` base class, if they aren't all critical. Maybe just `HarvestJobException`. And then rename the ones that are "Critical" to have that in the name.